### PR TITLE
Add Return to Cart link in Tribe Commerce

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/100253

This PR:
* adds a "return to cart" link in Tribe Commerce ticket forms; the link will show only if at least one ticket was added to the cart
* adds `wpunit` tests for it
* adds the `functional` suite to test the feature on a high level

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $criteria is never used | [Unmanaged.php:63](https://github.com/moderntribe/event-tickets/blob/9959e7dbfab3b4cf15139015b73783c2e845d606/src/Tribe/Commerce/PayPal/Cart/Unmanaged.php#L63)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/tpp.php [hunk 1]
```diff
@@ -140,6 +140,14 @@ class="tribe-ticket-quantity qty"
 			</tr>
 		<?php endif ?>
 
+		<?php if ( tribe( 'tickets.commerce.paypal.cart' )->has_items() ) : ?>
+            <tr>
+                <td colspan="5" class="tpp-add">
+					<?php include tribe( 'tickets.commerce.paypal' )->getTemplateHierarchy( 'tickets/tpp-return-to-cart' ); ?>
```
Comment (bordoni) [inline src/views/tickets/tpp.php]: Weird spacing.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $criteria is never used | [Unmanaged.php:63](https://github.com/moderntribe/event-tickets/blob/e840d85538cdf0090988b4871f85b22db1388a61/src/Tribe/Commerce/PayPal/Cart/Unmanaged.php#L63)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $criteria is never used | [Unmanaged.php:63](https://github.com/moderntribe/event-tickets/blob/600e9a89004e409c90db326dd65344435adaac2d/src/Tribe/Commerce/PayPal/Cart/Unmanaged.php#L63)

> via jenkins/codesniffer-phpcs

**Comment by lucatume** [comment]
There is not really a good reason why the functional test should fail but I will look into this tomorrow with fresh eyes. For the time being good to code review.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $criteria is never used | [Unmanaged.php:63](https://github.com/moderntribe/event-tickets/blob/087afa5135c2ea186f3f28ffbf690e55a703e2c0/src/Tribe/Commerce/PayPal/Cart/Unmanaged.php#L63)

> via jenkins/codesniffer-phpcs

**Comment by lucatume** [comment]
Tests fixed and passing, good to review.

File: src/Tribe/Commerce/PayPal/Cart/Unmanaged.php [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+	/**
+	 * Returns the name of the transient used by the cart.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $invoice_number
+	 *
+	 * @return string
+	 */
+	public static function get_transient_name(
```
Comment (bordoni) [inline src/Tribe/Commerce/PayPal/Cart/Unmanaged.php]: Can we put it into one line?

**Comment by lucatume** [comment]
@bordoni not sure this should be merged into `master`; any particular branch I should target (or create)?

**Comment by lucatume** [comment]
Ready to merge but on hold waiting for the MR branch

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $criteria is never used | [Unmanaged.php:63](https://github.com/moderntribe/event-tickets/blob/1dcadf136ae9a188a5fb51af3f09f6704b59f7ae/src/Tribe/Commerce/PayPal/Cart/Unmanaged.php#L63)

> via jenkins/codesniffer-phpcs

**Pull request merged**